podcasts/Sharing/SharingModal.swift

```diff
@@ -255,3 +281,17 @@ fileprivate extension Podcast {
         return L10n.paidPodcastReleaseFrequencyFormat(frequency)
     }
 }
+
+fileprivate extension String {
```

Comment:
Do you think this can be re-used in other areas? Just thinking if we should move this to a generic extension, like `String+RegEx`.